toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapViewProxy.kt

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright 2023 Esri
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+package com.arcgismaps.toolkit.geocompose
+
+import com.arcgismaps.geometry.Point
+import com.arcgismaps.mapping.view.ScreenCoordinate
+
+
+/**
+ * Used to perform operations on a [MapView].
+ *
+ * There should be a one-to-one relationship between a [MapViewProxy] and a composable [MapView].
+ * Operations can only be performed once the component has entered the composition.
```

Comment:
I think we need to be a bit more specific about the behaviour with reagrd to composition state:
```suggestion
 * Operations can only be performed once the associated composable MapView has entered the composition. Operations performed when the associated composable MapView is not in the compositio will fail gracefully, i.e. won't throw exceptions but won't return a successful result.
```